### DISCUSSION TITLE
An Invitation to a Team Leads Meeting - Saturday, 26th October, 2024, 4-5Pm UTC!

### DISCUSSION THREAD

**Discussion by Rayccheal** [clentafrica/hr-people-operations#16] [Announcements]
Dear Team Leads, @clentafrica/wg-design @clentafrica/wg-disability-gender-equality @clentafrica/wg-education @clentafrica/wg-education @clentafrica/wg-energy @clentafrica/wg-sustainability-esg @clentafrica/wg-marketing-comms @clentafrica/wg-community-managers @clentafrica/wg-partnerships @clentafrica/wg-health @clentafrica/wg-web-dev @clentafrica/wg-fundraising @clentafrica/wg-hr @clentafrica/wg-health 

I hope you are doing well today. 

This is to invite you to a crucial Team Leads meeting this Saturday, October 26, 2024, from 4:00 PM to 5:00 PM UTC. 

Below are key topics that we hope to discuss;

- Follow-up on action points from previous meetings
- Emphasizing team structure and operations at CLENT Africa
- Proposed team projects
- HR projects and scheduling for upcoming meetings etc.

Your participation in this meeting is very crucial, as we aim to chart our course for future endeavors. Please confirm your local time zones[ here](https://www.worldtimebuddy.com/) and let us know if the proposed date and time is convenient for you.

I look forward to our discussion!

Best regards,
Racheal
HR Team Lead

**Comment by EmmanetD** [comment]
The date and time is convenient for me. Thank you @Rayccheal .
